modules/network-observability-viewing-alerts.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-alert-dashboard_{context}"]
+= Viewing health information
+
+You can access metrics about health and resources use of the Network Observability Operator from the *Dashboards* page in the web console. A health alert banner that directs you to the dashboard might sometimes appear on the *Network Traffic* and *Home* pages. Alerts are generated in the following cases:
+
+* The *NetObservLokiError* alert occurs if the `flowlogs-pipeline` workload is dropping flows because of Loki errors, such as if the Loki ingestion rate limit has been reached.
+* The *NetObservNoFlows* alert occurs if no flows are ingested for a certain amount of time.
+
+.Prerequisites
+
+* You have the Network Observability Operator installed.
+* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Dashboards*.
+. From the *Dashboards* dropdown, select *Netobserv/Health*. 
+. Metrics about the health of the Operator are displayed on the page. 
+
+
+== Disable health alerts
+Health alerts are automatically enabled. You can opt out of health alerting by editing the YAML:
```

Comment:
Good observation. I was considering putting this information in the Configuration area, but decided to keep it here instead. In that section, there's a procedure that shows how to edit the FlowCollector in the web console. But since I'm including this here, I need to add those steps. Thanks!